### ISSUE TITLE
[BUG] Unable to use compodoc with Apple Silicon M1 Chip

### ISSUE THREAD

**Issue by koen-dev** [compodoc/compodoc#1172]
##### **Overview of the issue**
Unable to use `compodoc` on M1 MacBook Pro. Not 100% sure if this relates to the Apple Silicon but as it seems to work on other Intel based MacBooks I assume it is.
![image](https://user-images.githubusercontent.com/5583604/148735165-4eb9204d-dfe6-4191-8056-613587077273.png)

##### **Operating System, Node.js, npm, compodoc version(s)**
- Operating System: **macOs Monterey (12.1)**
- Chip: **Apple M1 Pro**
- Node.js: **14.18.2**
- npm: **6.14.15**
- compodoc: **1.1.16**

##### **Angular configuration, a `package.json` file in the root folder**

```json
{
  "name": "ngx-storybook",
  "private": true,
  "version": "0.0.120",
  "scripts": {
    "start": "npm run storybook",
    "prebuild": "rm -rf dist",
    "build": "NODE_ENV=production build-storybook -o dist",
    "build:watch": "exit 0",
    "test": "exit 0",
    "docs:json": "compodoc -p ./tsconfig.json -e json -d .",
    "storybook": "npm run docs:json && start-storybook -p 6006"
  },
  "peerDependencies": {
    "@angular/cdk": "13.x.x",
    "@angular/common": "13.x.x",
    "@angular/core": "13.x.x",
    "@angular/forms": "13.x.x",
    "@angular/platform-browser": "13.x.x",
    "@angular/animations": "13.x.x",
    "@angular/router": "13.x.x"
  },
  "dependencies": {
    "tslib": "~2.1.0",
    "zone.js": "~0.11.4"
  },
  "devDependencies": {
    "@angular/animations": "~13.0.0",
    "@angular/common": "~13.0.0",
    "@angular/compiler": "~13.0.0",
    "@angular/core": "~13.0.0",
    "@angular/forms": "~13.0.0",
    "@angular/router": "~13.0.0",
    "@angular/platform-browser": "~13.0.0",
    "@angular/platform-browser-dynamic": "~13.0.0",
    "@angular-devkit/build-angular": "~13.0.0",
    "@schiphol-ac/ngx-ac": "workspace:*",
    "@angular/cli": "~13.0.0",
    "@angular/compiler-cli": "~13.0.0",
    "@angular/elements": "^13.0.2",
    "@babel/core": "7.6.0",
    "@compodoc/compodoc": "1.1.16",
    "@storybook/addon-docs": "6.4.8",
    "@storybook/addon-actions": "6.4.8",
    "@storybook/addon-essentials": "6.4.8",
    "@storybook/addon-links": "6.4.8",
    "@storybook/angular": "6.4.8",
    "@storybook/builder-webpack5": "^6.4.8",
    "@storybook/manager-webpack5": "^6.4.8",
    "@types/jasmine": "~3.10.0",
    "@types/node": "14.17.3",
    "@webcomponents/custom-elements": "^1.5.0",
    "babel-loader": "8.2.3",
    "typescript": "~4.4.4",
    "@rushstack/heft": "0.41.7",
    "@types/heft-jest": "~1.0.2",
    "@types/jest": "~26.0.20",
    "jest": "~26.6.3",
    "ts-node-dev": "^1.1.8",
    "ts-jest": "26.4.4",
    "ng-packagr": "~13.0.0",
    "rxjs": "6.5.5",
    "tsconfig-paths-webpack-plugin": "~3.5.1"
  }
}
```

##### **Compodoc installed globally or locally ?**
Locally

##### **If possible your terminal logs before the error**

![image](https://user-images.githubusercontent.com/5583604/148735165-4eb9204d-dfe6-4191-8056-613587077273.png)

##### **Reproduce the error**

Do the following on a Apple Silicon Chip:
- `compodoc -p ./tsconfig.json -e json -d .`

##### **Related issues**

#1050

**Comment by vogloblinsky** [comment]
Update compodoc to 1.1.17 version, which use a fixed version of colors, which is the origin of this error.

**Comment by koen-dev** [comment]
Awesome, thanks for your quick response! Going to test it right now. Will update you if it works 👍

**Comment by koen-dev** [comment]
@vogloblinsky I updated the package to 1.1.17 and I still have the same issue unfortunately 😢

**Comment by vogloblinsky** [comment]
I am on it

**Comment by vogloblinsky** [comment]
Ok, fixed with compodoc 1.1.18
